### PR TITLE
Static WebGL image.

### PR DESCRIPTION
**Pull request by Lokathor**
Part of https://github.com/rust-tutorials/triangle-from-scratch/issues/8

Doesn't do user interaction, that'll be a separate page.

---
TODO:
* [x] move the `canvas` into the body tag.
* [x] Fix the "can't use `i64` note". Wasm is okay with `i64`, but the problem is that the FFI layer doesn't support it properly.

### CODE REVIEW

File: book_src/web_stuff/web_gl_with_bare_wasm.md [hunk 1]
```diff
@@ -0,0 +1,683 @@
...
+## The Web Page
+
+Okay now we need a webpage for the user to display and have the wasm go.
+
+I'm absolutely not a web development person,
+but I know just enough to throw some HTML together by hand:
+
+```html
+<html>
+<canvas width="800" height="600" id="my_canvas"></canvas>
```
Comment (17cupsofcoffee) [inline book_src/web_stuff/web_gl_with_bare_wasm.md]: The `canvas` should go in the `body`, ideally. Web browsers are fairly generous about what weird and wonderful HTML they'll accept, though, so it wouldn't surprise me if this still worked.

File: book_src/web_stuff/index.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+If you want to run Rust code in a browser you compile it to [WebAssembly](https://en.wikipedia.org/wiki/WebAssembly),
+or WASM for short,
+which is an output target the same as compiling for windows x86_64,
+or linux arm,
+or any other target.
+
+Even then, Wasm is strongly sandboxed, and it cannot directly interact with the world.
+Not only do you have to bind to some external functions on the Rust side,
+you have to *write those eternal functions yourself* in javascript.
```
Comment (17cupsofcoffee) [inline book_src/web_stuff/index.md]:
```suggestion
you have to *write those external functions yourself* in javascript.
```

**Review by 17cupsofcoffee — COMMENTED** [review]
:+1: Looks good, the raw WASM approach wasn't nearly as arduous as I thought it'd be.

File: book_src/web_stuff/web_gl_with_bare_wasm.md (hunk 1 above)
Comment (Lokathor) [reply book_src/web_stuff/web_gl_with_bare_wasm.md]: Ah, yeah, "it works on my machine", and that's just how it was in the example i followed.

But we can move it into the body.

**Pull request merged**